packages/components/src/molecules/ValidationOverview/ProblemRenderer.tsx

```diff
@@ -31,6 +31,9 @@ const ProblemRenderer: React.FC<IProps> = props => {
     [selectedProblem, node.problem, groupByFilterValue]
   );
 
+  const absent = useMemo(() => node.problem.baselineState === 'absent', [node]);
+  const hasFix = useMemo(() => node.problem.fixes && node.problem.fixes.length > 0, [node]);
```

Comment:
I think we're actually better off without `useMemo` here

Reply:
I always doubt a bit about it; we're better off because the values are primitives so no re-renders on children are triggered when this rerenders as long as the primitive value does not change?